### ISSUE TITLE
无法从 athena-os.github.io : The requested URL returned error: 404 获取文件 'athena-repository.db'

### ISSUE THREAD

**Issue by tzyyyyyyy** [Athena-OS/athena-repository#8]
我在用pacman更新时  无法同步更新数据库 并用安装每个工具的时候都会提示：警告：database file for 'athena-repository' does not exist (use '-Sy' to download)  请问我该如何解决？
![微信图片_20240111114054](https://github.com/Athena-OS/athena-repository/assets/118863363/ab5d57f6-dbd3-426c-bb26-59245277660a)
![微信图片_20240111114100](https://github.com/Athena-OS/athena-repository/assets/118863363/9cfed5d1-a70a-4567-a097-6545e35fc465)

**Comment by D3vil0p3r** [comment]
You are using an old ISO

**Comment by tzyyyyyyy** [comment]
> You are using an old ISO
我更换了新版的iso，并且重新安装了 但是还是报错和数据库相关的问题
![微信图片_20240112092915](https://github.com/Athena-OS/athena-repository/assets/118863363/1a8a0c6f-5bd5-4c8d-8395-118b64a6cd4a)

**Comment by D3vil0p3r** [comment]
Did you ever use an Arch Linux distro?

**Comment by tzyyyyyyy** [comment]
> 您使用过 Arch Linux 发行版吗？

很抱歉，我用过unbuntu debian centos，还没有用过arch linux

**Comment by tzyyyyyyy** [comment]
我对照你之前回答的一些问题 解决了数据库的报错问题 但是现在还有一个报错
![微信图片_20240112100005](https://github.com/Athena-OS/athena-repository/assets/118863363/6070ef0e-681e-44de-92d6-4ecbc34303f7)

**Comment by D3vil0p3r** [comment]
First you should avoid to use root account for activities, because not secure. Then, if you get issues related to single mirrors, you can comment them in the mirrorlist files.

**Comment by Gravel-1019** [comment]
> 我对照你之前回答的一些问题 解决了数据库的报错问题 但是现在还有一个报错 ![微信图片_20240112100005](https://private-user-images.githubusercontent.com/118863363/296103711-6070ef0e-681e-44de-92d6-4ecbc34303f7.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDU4Mzc2MzMsIm5iZiI6MTcwNTgzNzMzMywicGF0aCI6Ii8xMTg4NjMzNjMvMjk2MTAzNzExLTYwNzBlZjBlLTY4MWUtNDRkZS05MmQ2LTRlY2JjMzQzMDNmNy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMTIxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDEyMVQxMTQyMTNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xODlmMzc1YTZiMzA2N2IzZDQwMDkyMDY0MTE1ZmE1OTlhMzVjOWM2MjIyNTE1YjczNDE3Y2QxODkzZTc4Y2MzJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.MDyYYJhSw0JaeRK9KAFpE71rVRW9vHPtdsEvXP5BZjE)

哥们，你之前无法从 athena-os.github.io : The requested URL returned error: 404 获取文件 'athena-repository.db'这个问题是咋解决的，我解决不了，换了源也没用

**Comment by D3vil0p3r** [comment]
* Solution: edit `/etc/pacman.conf` and replace `[athena-repository]` by `[athena]`. Then replace the content of `/etc/pacman.d/athena-mirrorlist` by:
```
# Global
Server = https://athena-os.github.io/$repo-repository/$arch

# Lithuania
Server = https://hub.athenaos.org/$repo/$arch
```

**Comment by Gravel-1019** [comment]
> * 解决方案：编辑并替换为 。然后将以下内容替换为：`/etc/pacman.conf``[athena-repository]``[athena]``/etc/pacman.d/athena-mirrorlist`
> 
> ```
> # Global
> Server = https://athena-os.github.io/$repo-repository/$arch
> 
> # Lithuania
> Server = https://hub.athenaos.org/$repo/$arch
> ```

I replaced the contents of "/etc/pacman.d/athena-mirrorlist" as you said, but when I execute "pacman -syu", it still prompts: "athena-repository .db download failed".Could it be because I'm in China and can't access foreign networks due to regional network restrictions?

**Comment by D3vil0p3r** [comment]
Did you replace the string also in `/etc/pacman.conf/ file?

**Comment by Gravel-1019** [comment]
Yes, I replace the string in file /etc/pacman.conf. But it still failed to get  athena-repository

**Comment by D3vil0p3r** [comment]
Maybe because you are using `s` lowercase in pacman command. Run `sudo pacman -Syyu`.

**Comment by Gravel-1019** [comment]
I tried using the latest system version, but when I clicked the reboot button, the screen went black。。。

**Comment by D3vil0p3r** [comment]
but it reboots? Are you in baremetal or VM? Which environment are u using (GNOME, XFCE, etc.)?

**Comment by Gravel-1019** [comment]
I am in VM. And I use GNOME

**Comment by D3vil0p3r** [comment]
Wayland or Xorg?

**Comment by Gravel-1019** [comment]
It should be Xorg

**Comment by D3vil0p3r** [comment]
what if you open the terminal and run `sudo reboot`?

**Comment by Gravel-1019** [comment]
When I turn on the system, it doesn't show any text, but gets stuck on a black screen

**Comment by D3vil0p3r** [comment]
Currently we are working on migration from Arch to Nix. I suggest you to wait until our pre-release and try it.

**Comment by Gravel-1019** [comment]
Thank you for your answer. In my opinion, Athena OS is a great operating system, the UI is very cool. Hopefully, the Athena system will be a success！

**Comment by Gravel-1019** [comment]
> 实际上出现黑屏是因为其实根本没安装上，如果我没猜错你用的新版应该只有在线安装吧，然后因为国内网络的原因，这东西根本下不下来，然后就报了个error然后就跳到下一环节直接让你reboot了，我这里之前是这种情况，所以用了旧版的iso，因为有离线安装。 @Gravel-1019

我也尝试过旧版的安装包，可是升级软件包的时候有一个database的文件总是装不上

**Comment by D3vil0p3r** [comment]
> > 实际上出现黑屏是因为其实根本没安装上，如果我没猜错你用的新版应该只有在线安装吧，然后因为国内网络的原因，这东西根本下不下来，然后就报了个error然后就跳到下一环节直接让你reboot了，我这里之前是这种情况，所以用了旧版的iso，因为有离线安装。 @Gravel-1019
> 
> 我也尝试过旧版的安装包，可是升级软件包的时候有一个database的文件总是装不上

It was one of the reasons we switched to NixOS

**Comment by 404lol13** [comment]
> * Solution: edit `/etc/pacman.conf` and replace `[athena-repository]` by `[athena]`. Then replace the content of `/etc/pacman.d/athena-mirrorlist` by:
> 
> ```
> # Global
> Server = https://athena-os.github.io/$repo-repository/$arch
> 
> # Lithuania
> Server = https://hub.athenaos.org/$repo/$arch
> ```

is there any server in china?foreign server is not available in china mainland.
the same problem with athena installed by nixos,the VM machine cant link to github

**Comment by D3vil0p3r** [comment]
> > * Solution: edit `/etc/pacman.conf` and replace `[athena-repository]` by `[athena]`. Then replace the content of `/etc/pacman.d/athena-mirrorlist` by:
> > 
> > ```
> > # Global
> > Server = https://athena-os.github.io/$repo-repository/$arch
> > 
> > # Lithuania
> > Server = https://hub.athenaos.org/$repo/$arch
> > ```
> 
> is there any server in china?foreign server is not available in china mainland. the same problem with athena installed by nixos,the VM machine cant link to github

Mmm what about GitLab in China?
GitHub is not accessible from China?
